test.js

```diff
@@ -79,7 +79,7 @@ test('should return warning if repository field is missing', verifyResult, {
 	},
 });
 
-test.failing('should support user/repo format', verifyResult, {
+test('should support user/repo format', verifyResult, {
```

Comment:
Would be good to have a test for `gitHub:user/repo` and `user/repo#branch`

Reply:
Thanks. Just find that `github-url-from-git` doesn't match any suffix path from a GitHub repo URL.. I've updated the logic to support that.